### PR TITLE
Pose limits on the tx pool size and the per-account tx pool size.

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Pose limits on the tx pool size and the per-account tx pool size

Related issue: https://github.com/ainblockchain/ain-blockchain/issues/316

### CODE REVIEW

**Review by liayoo — APPROVED** [review]
LGTM, thanks! No more bloated tx pool! 🙌

File: integration/node.test.js [hunk 1]
```diff
@@ -2346,6 +2349,77 @@ describe('Blockchain Node', () => {
...
+            signature,
+          });
+        }
+        const res1 = await client.request('ain_sendSignedTransactionBatch', {
+          tx_list: txList1,
+          protoVer: CURRENT_PROTOCOL_VERSION
+        });
+        const resultList1 = _.get(res1, 'result.result', null);
+        // Accepts transactions.
+        expect(Array.isArray(resultList1)).to.equal(true);
```
Comment (minsulee2) [inline integration/node.test.js:2385]: Q) We have got Chainutil.isArray() though, Do we use Array.isArray() for the test case only?

**Review by minsulee2 — APPROVED** [review]
LGTM with a question!

File: integration/node.test.js (hunk 1 above)
Comment (platfowner) [reply integration/node.test.js]: I think it's totally fine in tests (as they're the same internally) but tried to update them. Thanks!

**Comment by platfowner** [comment]
Thanks for the fast reviews! Now merging..

**Pull request merged**